lib/loaders/loaders_without_authentication/gravity.js

```diff
@@ -2,9 +2,10 @@
 import factories from "../api"
 
 export default requestID => {
-  const { gravityLoaderWithoutAuthenticationFactory } = factories(requestID)
-  const gravityLoader = gravityLoaderWithoutAuthenticationFactory
-
+  const {
+    gravityLoaderWithoutAuthenticationFactory: gravityLoader,
+    gravityLoaderWithoutAuthenticationWithoutCachingFactory: oneOffGravityLoader,
```

Comment:
Here’s a good example of where the ambiguity and naming leads us astray. Data loaders _should_ usually ‘cache’, just maybe only for the duration of the query execution and **not** to memcache.

Reply:
Hmm, I'm not sure I follow. Can you expand on this? Or are there some docs/blogs I should familiarize myself with?

Reply:
For the duration of the query being executed, you don’t ever want to make more than 1 request to the Gravity endpoint, right? So its value should be cached for that short period in time, but not stored in memcache and thus shared with other queries being executed.

Reply:
Maybe this helps https://github.com/facebook/dataloader#creating-a-new-dataloader-per-request

Reply:
Cool, that makes a lot of sense, thanks!